aldryn_config.py

```diff
@@ -368,10 +377,32 @@ def storage_settings_for_media(self, settings, env):
         settings['MEDIA_ROOT'] = env('MEDIA_ROOT', os.path.join(settings['DATA_ROOT'], 'media'))
         settings['MEDIA_HEADERS'] = []
 
-    def storage_settings_for_static(self, settings, env):
+        cmds = {}
+        if os.path.exists('/usr/bin/pngout'):
+            cmds['png'] = '/usr/bin/pngout {filename} {filename}.png -s0 -y && mv {filename}.png {filename}'
+        if os.path.exists('/usr/bin/jpegoptim'):
+            cmds['jpeg'] = '/usr/bin/jpegoptim --max=90 --overwrite --strip-all --all-progressive {filename}'
+        if os.path.exists('/usr/bin/gifsicle'):
+            cmds['gif'] = '/usr/bin/gifsicle --batch --optimize=2 {filename}'
+        settings['THUMBNAIL_OPTIMIZE_COMMAND'] = cmds
```

Comment:
@stefanfoulis what about setting that setting here instead of `STATIC_IMAGES_OPTIMIZE_COMMAND`?

See the logic in https://github.com/aldryn/aldryn-django/pull/59/files#diff-c6740ec61ea2b22825c0abbf26b954e6R19 to select what is read.